logic/round.js

```diff
@@ -245,17 +245,19 @@ Round.prototype.applyRound = function() {
 			changes,
 		});
 
+		const accountData = {
+			publicKey: delegate,
+			balance: self.scope.backwards ? -changes.balance : changes.balance,
+			u_balance: self.scope.backwards ? -changes.balance : changes.balance,
+			blockId: self.scope.block.id,
+			round: self.scope.round,
+			fees: self.scope.backwards ? -changes.fees : changes.fees,
+			rewards: self.scope.backwards ? -changes.rewards : changes.rewards,
+		};
+
 		p = new Promise((resolve, reject) => {
 			self.scope.modules.accounts.mergeAccountAndGet(
-				{
-					publicKey: delegate,
-					balance: self.scope.backwards ? -changes.balance : changes.balance,
-					u_balance: self.scope.backwards ? -changes.balance : changes.balance,
-					blockId: self.scope.block.id,
-					round: self.scope.round,
-					fees: self.scope.backwards ? -changes.fees : changes.fees,
-					rewards: self.scope.backwards ? -changes.rewards : changes.rewards,
-				},
+				accountData,
```

Comment:
Given the issue here is `no-loop-func`, this seems like an irrelevant change. Is this a bug in ESLint?

Reply:
yes this was a eslint error and after moving that a variable declaration the error went off.